src/modules/10-common/featureFlags.ts

```diff
@@ -29,6 +29,7 @@ export enum FeatureFlag {
   FFM_1827 = 'FFM_1827',
   FFM_3959_FF_MFE_Environment_Detail = 'FFM_3959_FF_MFE_Environment_Detail',
   FFM_5939_MFE_TARGET_GROUPS_LISTING = 'FFM_5939_MFE_TARGET_GROUPS_LISTING',
+  FFM_6666_FF_MFE_Target_Groups_Detail = 'FFM_6666_FF_MFE_Target_Groups_Detail',
```

Comment:
Groups shouldn't be pluralised since it's just one we're wanting
```suggestion
  FFM_6666_FF_MFE_Target_Group_Detail = 'FFM_6666_FF_MFE_Target_Group_Detail',
```

Reply:
done